sidebars.js

```diff
@@ -1,28 +1,22 @@
 module.exports = {
   someSidebar: {
-//    '🐣 Start Here': [
-//      'introduction',
-//      'start-here/for-players',
-//      'start-here/for-guilds',
-//      'start-here/for-patrons',
-//    ],
-    '🤨 WTF is MetaGame?': [
+    'enter-metagame/signpost',
```

Comment:
Docusaurus seems to need to have the ` 'label': ['path']` structure. And i didn't seem anything in the docs about adding top level, childless sidebar items. It's going to need more thinking about or go down the route of building a Gatsby site and use Sanity.io for the CMS.

Reply:
I've changed this in a branch and will push to this PR but I invite discussion on what to do here.